_includes/js/getting-started.md

```diff
@@ -11,3 +11,9 @@ The Parse platform provides a complete backend solution for your mobile applicat
 Our JavaScript SDK is originally based on the popular [Backbone.js](http://backbonejs.org/) framework, but it provides flexible APIs that allow it to be paired with your favorite JS libraries. Our goal is to minimize configuration and let you quickly start building your JavaScript and HTML5 app on Parse.
 
 Our SDK supports Firefox 23+, Chrome 17+, Safari 5+, and IE 10. IE 9 is supported only for apps that are hosted with HTTPS.
+
+To initialize your own Parse-Server with Javascript, you should replace your curently initialize code with this
+<pre><code class="javascript">
```

Comment:
I believe you can use triple backticks here. The format you're using is only necessary in cases where the included code has &lt; or &gt; which sometimes messes up the renderer.